prime-router/docs/api/waters-reports.yml

```diff
@@ -11,7 +11,7 @@ paths:
     post:
       description: submit a report to be routed
       security:
-        - OktaAuth: [ FUNCTION ]
+        - OAuth2: [ system_admin ]
       parameters:
```

Comment:
Is there a "least privilege" alternative? There should be.

Reply:
The actual permissions used/required needs review across all endpoints.

This should be address separately.